src/gate_detect/test/testUtilities/TestUtils.h

```diff
@@ -0,0 +1,20 @@
+//
+// Created by da-cam on 15/06/18.
+//
```

Comment:
Should have standard start of file comment.

Reply:
Added

Reply:
Done